### PR TITLE
Issue #2908430 by WidgetsBurritos: Global Configuration Settings

### PR DESCRIPTION
**Pull request by WidgetsBurritos**
Drupal.org Issue: https://www.drupal.org/project/web_page_archive/issues/2908430

This PR introduces global configuration settings page to manage default settings for config entities and capture utilities. The functionality mostly works. I still need to add in functional tests for this which is why it's still WIP, but opening the PR just to get an initial round of tests going.

### CODE REVIEW

File: modules/wpa_screenshot_capture/config/schema/wpa_screenshot_capture.schema.yml [hunk 1]
```diff
@@ -17,3 +17,26 @@ web_page_archive.capture_utility.wpa_screenshot_capture:
     delay:
       type: integer
       label: 'Delay'
+
+web_page_archive.wpa_screenshot_capture.settings:
+  type: config_object
+  label: 'HTML Capture Utility settings'
```
Comment (bighappyface) [inline modules/wpa_screenshot_capture/config/schema/wpa_screenshot_capture.schema.yml]: Mislabeled?

File: modules/wpa_skeleton_capture/config/schema/web_page_archive_example.schema.yml [hunk 2]
```diff
@@ -5,3 +5,15 @@ web_page_archive.capture_utility.wpa_skeleton_capture:
     capture:
       type: int
       label: 'Width'
+
+
+web_page_archive.wpa_skeleton_capture.settings:
+  type: config_object
+  label: 'HTML Capture Utility settings'
```
Comment (bighappyface) [inline modules/wpa_skeleton_capture/config/schema/web_page_archive_example.schema.yml]: Mislabeled?

**Review by bighappyface — COMMENTED** [review]
Looking really good

**Comment by WidgetsBurritos** [comment]
@bighappyface Thanks for the feedback.  Fixed.

**Review by bighappyface — APPROVED** [review]
+1 for d53253a5c695dbb8c021b0d7cec1900c6c879490

**Pull request merged**